js-exercises/object-utils/objectUtils.js

```diff
@@ -0,0 +1,28 @@
+function getKeyDescriptorPairs(object) {
+  return Object.getOwnPropertyNames(object).map((key) => {
+    const desc = Object.getOwnPropertyDescriptors(object, key);
+    return [key, desc];
```

Comment:
instead of getOwnPropertyDescriptors() method you should have used getOwnPropertyDescriptor().

Also, the getOwnPropertyDescriptors() method does not take second argument which is key here rather it takes the object and returns an object with all the descriptors